core/sql/cli/Cli.cpp

```diff
@@ -8554,17 +8558,34 @@ Lng32 SQLCLI_LOBcliInterface
     case LOB_CLI_INSERT:
       {
 	// insert into lob descriptor handle table
-	str_sprintf(query, "select syskey from (insert into table(ghost table %s) values (%ld, 1, %ld)) x",
-		    lobDescHandleName, descPartnKey, (dataLen ? *dataLen : 0));
+        str_sprintf(query, "select syskey from (insert into table(ghost table %s) values (%ld, 1, %ld)) x",
+                    lobDescHandleName, descPartnKey, (dataLen ? *dataLen : 0));
+                /*str_sprintf(query, "select syskey from (insert into table(ghost table cast(? as varchar(1024)) values (cast(? as largeint), 1, cast(? as largeint))) x");*/
+		   
         lobDebugInfo(query,0,__LINE__,lobTrace);
 	// set parserflags to allow ghost table
 	currContext.setSqlParserFlags(0x1);
 	
-	Int64 descSyskey = 0;
+        Int64 descSyskey = 0;
 	Lng32 len = 0;
-	cliRC = cliInterface->executeImmediate(query,
+        /*
+        strcpy(stmtName,"INS1");
+        cliRC = cliInterface->executeImmediatePrepare(query,NULL,0,&rowsAffected,FALSE,stmtName);
+	if (cliRC < 0)
+	    goto error_return;
+        char *inputValues = new (currContext.exHeap()) char[sizeof(Int32) + strlen(lobDescHandleName) + sizeof(Int64)*2];
+        *(Int32 *)inputValues= strlen(lobDescHandleName);
+        memcpy(&inputValues[sizeof(Int32)],lobDescHandleName,strlen(lobDescHandleName));
+       
+        inputValues[sizeof(Int32) + strlen(lobDescHandleName)] = descPartnKey;
+        inputValues[sizeof(Int32) + strlen(lobDescHandleName)+sizeof(Int64)] = (dataLen ? *dataLen : 0);
+        inputValuesLen = sizeof(Int32) + strlen(lobDescHandleName)+sizeof(Int64)*2;
+        cliRC = cliInterface->clearExecFetchCloseOpt((char *)inputValues,inputValuesLen,(char*)&descSyskey,&len, &rowsAffected);
+        */
```

Comment:
Why was this block of commented-out code added?

Reply:
This was from the source branch when a new method of saving prepared statements was attemtped. Removing it since it's being reworked.